__test__/unitTest/removeInvalidTokensFromList.test.ts

```diff
@@ -0,0 +1,33 @@
+// import { removeInvalidTokensFromList, validateTokenListWithErrorThrowing, tokenListIsValid} from '../../src/lib/utils'
+import { removeInvalidTokensFromList } from "../../src/lib/utils"
+import arblist from "./schma/arbify.tokenlist.json";
+import arblistdecimalsTooLow from "./schma/arbifyInvalid1.tokenlist.json"
+import arblistWrongVersion from "./schma/arbifyInvalid7.tokenlist.json"
+
+describe("removeInvalidTokensFromList Test", () => {
+    it("Should return same when use correct list", () => {
+        
+        expect(removeInvalidTokensFromList(arblist)).toEqual(arblist)
+    })
+
+    it("Should remove wrong token when use incorrect list", () => {
+        const preImage = JSON.parse(
+            JSON.stringify(arblistdecimalsTooLow)
+          ) as typeof arblistdecimalsTooLow;
+        const correctList = removeInvalidTokensFromList(arblistdecimalsTooLow)
+        expect(correctList.tokens.length).toBeLessThan(preImage.tokens.length)
+        correctList.tokens.forEach(tokenInfo => {
+            expect(tokenInfo.name).not.toEqual("blah blah ")
+        })
+        arblistdecimalsTooLow.tokens = preImage.tokens
```

Comment:
Why are we assigning this here?

Reply:
This is for future development, thought it may reuse arblistdecimalsTooLow after, but yea, we can remove it since this is useless now

Reply:
Oh, we can change a way to handle this. (doesn't copy to `preimage`, but copy to `listToBeFixed`)